infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -121,13 +121,18 @@ private int invalidateSmallestTxVersion() {
 
         int numEntries = 0;
         while (firstAddress() == firstEntry.txVersion) {
-            log.debug("evict items " + numEntries + " with address " + firstAddress());
+            if (log.isTraceEnabled()) {
```

Comment:
is it necessary to check if trace is enabled?  The message will not get logged if logging level is > trace

Reply:
@pankti-m 
https://stackoverflow.com/questions/9520093/why-is-it-necessary-to-use-the-istraceenabled-function-in-jboss